src/cheri/riscv-unpriv-integration.adoc

```diff
@@ -801,9 +797,10 @@ include::insns/store_32bit_cap.adoc[]
 
 {cheri_base_ext_name} extend existing instructions that are used for handling addresses so that they manipulate a whole capability.
 
-* The affected operands change type from *x* to *c* to represent this.
-* Whenever the address field of any capability is modified, it is _always_ updated using <<SCADDR>> semantics.
+* The affected operands change type from *x* to *c* to represent this. #ARC-COMMENT: What does this actually mean?  What type is x versus c?#
+* Whenever the address field of any capability is modified, it is _always_ updated using <<SCADDR>> semantics. #ARC-COMMENT: Capabilities are not modified.  Capabilites are either input source operands or results from an instruction, they are not modified in place by an instruction.  This should be stated as a rule on results of operations.#
 * Any instruction that writes back an address (e.g., <<AUIPC_CHERI>> or <<CSRRW_CHERI>>) to the destination register, writes a full capability register instead.
+ #ARC-COMMENT: Doesn't this rule apply to all instructions that operate on a capability?  Or are there instructions that will always convert result to a non-capability?  The rule should be stated more cleanly - the instructions don't know if values are addresses, and non-capability values can be addresses.#
```

Comment:
I guess we have some work to do here to specify result types

Reply:
Tried to clarify this now.

Reply:
What we have is good enough for now - it may change following review next week